content/docs/interface/example1/_index.md

```diff
@@ -110,11 +110,11 @@ run_metadata:
   default_input_namespace: soniamitchell
   default_output_namespace: soniamitchell
   write_data_store: /Users/SoniaM/datastore/
-  public: true
   local_repo: /Users/Soniam/Desktop/git/SCRC/SCRCdata
+  script: R -f inst/SCRC/scotgov_management/submission_script.R /Users/SoniaM/datastore/coderun/20210511-231444/
```

Comment:
For consistency, should this be:
```suggestion
  script: |-
    R -f inst/SCRC/scotgov_management/submission_script.R /Users/SoniaM/datastore/coderun/20210511-231444/
```

Reply:
yea, as a one line script I don't think it matters.. I've been needlessly copy and pasting the `|-`. For interest, it's a "chomp modifier" https://www.cloudbees.com/blog/yaml-tutorial-everything-you-need-get-started

Reply:
Yes, I was just meaning that if this is a working yaml, it'll always be produced the same way, so you should make them all the same - either on one line, with `|` or with `|-`. I like "chomp modifier" though!

Reply:
Good point! I’ll keep an eye out for that 👍